projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.ts

```diff
@@ -41,5 +43,6 @@ export class NgxRowAccordionComponent implements OnInit, OnDestroy {
 
   toggle() {
     this.ngxRowAccordionService.toggle(this.id);
+    this.status = !this.status;
```

Comment:
The status should not be defined from there.
It should be contained into the service and the view should be driven by the value coming from the service. This should be an observable.